spec/commands/proposals/create_proposal_spec.rb

```diff
@@ -0,0 +1,199 @@
+# frozen_string_literal: true
```

Comment:
Can you remove the tests non related to checking the `author_*` fields please. Otherwise this tests are hard to maintain.